pager.c

```diff
@@ -1710,8 +1710,10 @@ static int display_line(FILE *fp, LOFF_T *last_pos, struct Line **line_info,
     }
 
     /* this also prevents searching through the hidden lines */
-    if ((flags & MUTT_HIDE) && ((*line_info)[n].type == MT_COLOR_QUOTED))
+    if ((flags & MUTT_HIDE) && ((*line_info)[n].type == MT_COLOR_QUOTED) &&
+        ((*line_info)[n].quote->index >= C_ToggleQuotedShowLevels)) {
```

Comment:
`(*line_info)[n].quote` is `NULL` for non-quoted lines. That triggered a crash as soon as I run-tested this branch ;)

Reply:
Curious, I never saw any crashes, but OK, I can see it could happen depending on some variables set.

On the presumption that the value of quote and quote->index never mattered before, I'm
suggesting the following to behave as before if quote is NULL, and only evaluate 
the index if it is not NULL.

Does that sound OK? 

If so, how about this change:

```diff
diff --git a/pager.c b/pager.c
index 3881efd9326c..b1377725c317 100644
--- a/pager.c
+++ b/pager.c
@@ -1711,7 +1711,8 @@ static int display_line(FILE *fp, LOFF_T *last_pos, struct Line **line_info,

     /* this also prevents searching through the hidden lines */
     if ((flags & MUTT_HIDE) && ((*line_info)[n].type == MT_COLOR_QUOTED) &&
-        ((*line_info)[n].quote->index >= C_ToggleQuotedShowLevels)) {
+        (((*line_info)[n].quote == NULL) ||
+         ((*line_info)[n].quote->index >= C_ToggleQuotedShowLevels))) {
       flags = 0; /* MUTT_NOSHOW */
     }
   }
```